### DISCUSSION TITLE
Show overview page for main entries in sidebar

### DISCUSSION THREAD

**Discussion by Nadine2016** [colinwilson/lotusdocs#58] [Q&A]
Hi Colin, 

similar to the doc landing page where you show all the main nav entries from the sidebar as tiles 

<img width="1702" alt="image" src="https://github.com/colinwilson/lotusdocs/assets/17709352/89eb0f1a-c877-411c-bb0b-8689b88ec0aa">

Is there a way to configure the site to do that for other _index.md files, such as the Quickstart or Guides? We would love to have this type of overview page for all _index.md files to allow users to preview what is in a particular section. 

Thank you and best regards, 
Nadine

**Comment by colinwilson** [comment]
Do you mean like the [Guides list page](https://lotusdocs.dev/docs/guides/)?

![Screenshot 2023-09-14 at 17-00-02 Guides Lotus Docs](https://github.com/colinwilson/lotusdocs/assets/12916656/b86f4cb7-0599-4594-8810-b7fb4e2f041c)

**Reply by Nadine2016** [reply]
oh yeah, that is exactly what I am looking for. I did not realize that this was available because when you click on `Guides` this page does not show up. but when you type in the link directly it works.

**Reply by colinwilson** [reply]
Yep. 👍

Originally I was thinking of making the section menu items a [split dropdown button/function](https://getbootstrap.com/docs/5.3/components/dropdowns/#split-button) type. But ultimately went with the current functionality. I may revisit this at a later date though.

As you pointed out, you can link to those pages.
